scipy/optimize/cython_optimize/meson.build

```diff
@@ -0,0 +1,16 @@
+py3.install_sources(
+    ['c_zeros.pxd', 
+     '_zeros.pxd'],
+    pure: false,
+    subdir: 'scipy/optimize'
+)
+
+run_command(py3, [cythonize, '.'], check : true)
```

Comment:
We shouldn't use `cythonize`, it will invoke setuptools under the hood. This line doesn't seem used anyway?

Reply:
There is a file, `_zeros.pyx.in` which gets converted to `_zeros.pyx` with the help of `cythonize`. Then `meson` is able to handle `_zeros.pyx`. Is there any other way to deal with `_zeros.pyx.in`?

Reply:
Ah, this is similar to `scipy/linalg/_decomp_update.pyx.in`, see `linalg/meson.build`. That extension builds, and the same pattern should work for `zeros.pyx.in`. It may or may not pass the tests (not sure, may depend on that open Meson PR linked in the comment above `_decomp_update`), but that is preferable. Or just comment it out for now and add a TODO.